Makefile

```diff
@@ -1,6 +1,38 @@
 .ONESHELL:
 SHELL = /bin/bash
 
+ifneq ($(kernel_build_method), "build")
```

Comment:
How does one set `kernel_build_method`? Should this become a command-line parameter specified in rules/config?

Reply:
It is. Please take a look at the linked PR.

Reply:
Aha. Sorry, missed the linked PR.

A small concern is the terminology might not be intuitive (i.e., that one of the "build methods" is "download" which doesn't actually build anything). I think a name like "kernel_procure_method" is more accurate. Maybe someone else has another suggestion?

Reply:
procure is good. Thanks.